lib/LocalVerifiableCredentialStore.js

```diff
@@ -207,7 +230,18 @@ export class LocalVerifiableCredentialStore {
         }
       }
       const queryType = query.map(q => q.type);
-      return this.find({type: queryType});
+      const queryTypes = Array.isArray(queryType) ? queryType : [queryType];
+      const selector = {
+        'meta.profileId': this.profileId,
+        'content.type': {$elemMatch: {$in: queryTypes}}
```

Comment:
`$elemMatch` seems like it would require the types to be in the same order when checking. Does doing just the `$in` query not work? Is this supposed to only match where *all* the types are present? If so -- is there a better query that avoids the ordering issue?